data/MoNihal.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name" : "Mohammed Nihal" ,
+  "displayStatsPublic" :true ,
+  "type" : "personal" ,
+  "bio" : " cs undergrad!!! " ,
+  "avatar" : "https://github.com/MoNihal10.png",
+  "links" :[
+  {
+    "name" : "Follow me on Twitter" ,
+    "url" : "https://twitter.com/n_ihal_10"
```

Comment:
and a missing comma right here at the end of line 10

Reply:
I have made the changes.

Reply:
Now, do I have to close the pull request and reopen it??

Reply:
Nah it's already open, you don't need to do anything

Reply:
How long does it usually take to merge a pull request and get the website to start running in my custom url??

Reply:
> How long does it usually take to merge a pull request and get the website to start running in my custom url??

It depends, sometimes maintainers are busy with other things right
But it doesn't take more than a day, so you'll have it by tomorrow for sure

Reply:
Alright thanks!!